landing-page/content/common/spark-quickstart.md

```diff
@@ -0,0 +1,323 @@
+---
+title: "Spark and Iceberg Quickstart"
+weight: 100
+url: spark-quickstart
+aliases:
+    - "quickstart"
+    - "quickstarts"
+    - "getting-started"
+disableSidebar: true
+disableToc: true
```

Comment:
Since we include a TOC at the top of the quickstart, it felt odd seeing both. Would you rather have the fixed TOC on the right and remove it from the intro section?